### DISCUSSION TITLE
surfaceConitaner and surfaceContainerHighest do not change anything

### DISCUSSION THREAD

**Discussion by earlybirdsignals** [rydmike/flex_color_scheme#266] [Q&A] [answered]
Hello All,

I am using flex_color_scheme: ^7.3.1 and trying to change my textformfield fill color. I checked the examples on website and surfaceContainerHighest is used to fill these widgets however in my code, surfaceContainer and it's versions do not effect any color. All white for all theme models. Using blueWhale currently.

I guess, I am missing sth but I could not find any solution.

Material App:

```
child: MaterialApp(
        debugShowCheckedModeBanner: false,
        title: 'XXXXXX',
        themeMode: ThemeMode.system,
        theme: FlexColorScheme.light(scheme: FlexScheme.outerSpace, useMaterial3: true).toTheme,
        darkTheme: FlexColorScheme.dark(scheme: FlexScheme.blueWhale, useMaterial3: true).toTheme,
        home: myWidget,
      ),
```

TextFormField: -> `fillColor: Theme.of(context).colorScheme.surfaceContainerHighest,`

Thank you for your support.

Ergün

**Comment by rydmike** [comment, marked as answer]
Hi @earlybirdsignals, thank you for your question.

Version 7.3.1 does not support these colors, it was released before these colors existed in Flutter SDK so it knows nothing about them.

You will need to migrate to FlexColorScheme V8 to get support for all the new surface colors that were added in Flutter 3.22.0.

I recommend using latest stable **Flutter 3.24.5** and **FlexColorScheme 8.0.2**.

See https://docs.flexcolorscheme.com/whats_new

**Comment by rydmike** [comment]
Hi @earlybirdsignals, I'm going to close this question as answered, by converting it a Question and Answer and move it over to the repo discussion section, where other can find it too and might find it useful. If you have any questions on moving to V8, you are welcome to ask them in the discussions section too.
